### PR TITLE
Tutorial Translation in Marathi

### PR DESCRIPTION
**Pull request by abhaypawar**

### CODE REVIEW

**Comment by abhaypawar** [comment]
As the Tutorial is of some 1200 lines. This PR has work of about **120** lines translation work into Marathi. 
Please check the ability of the work to be merged into master branch. I will put a PR of next **100-200 lines** by tomorrow evening.
If any changes expected, You can let me know anytime.
Hope the maintainers assess the PR at **earliest.**
Thanks! and Happy Contributing!

File: content/tutorial/tutorial.md [hunk 1]
```diff
@@ -14,75 +14,76 @@ redirect_from:
 
 This tutorial doesn't assume any existing React knowledge.
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## ट्यूटोरियल सुरू करण्यापूर्वी {#before-we-start-the-tutorial}
 
-We will build a small game during this tutorial. **You might be tempted to skip it because you're not building games -- but give it a chance.** The techniques you'll learn in the tutorial are fundamental to building any React app, and mastering it will give you a deep understanding of React.
+या ट्यूटोरियल मध्ये आपण एक छोटासा गेम तयार करू.**आपण कदाचित येथे गेम तयार करण्यासाठी आला नसाल- परंतु एक संधी द्या.** आपण ट्यूटोरियलमध्ये शिकत असलेली टेक्निक कोणतेही अँप तयार करण्यासाठी मूलभूत आहेत आणि त्यामध्ये प्रभुत्व मिळविण्यामुळे आपल्याला प्रिऍक्टचे सखोल ज्ञान मिळेल.
```
Comment (visshaljagtap) [inline content/tutorial/tutorial.md:19]: Please use React only, don't convert it 
प्रिऍक्टचे -> React चे

Comment (abhaypawar) [reply content/tutorial/tutorial.md]: रिऍक्ट or React. What is preferred?
Please comment down below for guiding me!

**Comment by abhaypawar** [comment]
Please check the changes to line 19 : ( रिऍक्ट)

File: content/tutorial/tutorial.md (hunk 1 above)
Comment (visshaljagtap) [reply content/tutorial/tutorial.md]: React is preferred. first please check #3

Comment (abhaypawar) [reply content/tutorial/tutorial.md]: Thanks for correcting me. I request you to check the changes. I have followed the above guideline for contribution of translating work. 
Thanks again!

File: content/tutorial/tutorial.md [hunk 2]
```diff
@@ -16,37 +16,36 @@ This tutorial doesn't assume any existing React knowledge.
 
 ## ट्यूटोरियल सुरू करण्यापूर्वी {#before-we-start-the-tutorial}
 
-या ट्यूटोरियल मध्ये आपण एक छोटासा गेम तयार करू.**आपण कदाचित येथे गेम तयार करण्यासाठी आला नसाल- परंतु एक संधी द्या.** आपण ट्यूटोरियलमध्ये शिकत असलेली टेक्निक कोणतेही अँप तयार करण्यासाठी मूलभूत आहेत आणि त्यामध्ये प्रभुत्व मिळविण्यामुळे आपल्याला रिऍक्ट सखोल ज्ञान मिळेल.
+या ट्यूटोरियल मध्ये आपण एक छोटासा गेम तयार करू.**आपण कदाचित येथे गेम तयार करण्यासाठी आला नसाल- परंतु एक संधी द्या.** आपण ट्यूटोरियलमध्ये शिकत असलेली टेक्निक कोणतेही अँप तयार करण्यासाठी मूलभूत आहेत आणि त्यामध्ये प्रभुत्व मिळविण्यामुळे आपल्याला React चे सखोल ज्ञान मिळेल.
```
Comment (visshaljagtap) [inline content/tutorial/tutorial.md:19]: अँप -> अ‍ॅप

File: content/tutorial/tutorial.md [hunk 3]
```diff
@@ -16,37 +16,36 @@ This tutorial doesn't assume any existing React knowledge.
...
 
 या ट्यूटोरियलमधून मूल्य मिळविण्यासाठी आपल्याला एकाच वेळी सर्व विभाग पूर्ण करण्याची आवश्यकता नाही. आपण शक्य तितके पुढे वाचत जाण्याचा प्रयत्न करा - ते एक किंवा दोन विभाग असले तरीही चालतील.
 
 ### आपले काय बनवण्याचे उद्दिष्ट आहे? {#what-are-we-building}
 
-ह्या ट्युटोरिअल मध्ये आपण एक इंटरॅक्टिव्ह टिक-टॅक-तोव गेम, रिऍक्ट वापरून कसे बनवावे, हे पाहुयात.
+ह्या ट्युटोरिअल मध्ये आपण एक इंटरॅक्टिव्ह टिक-टॅक-तोव गेम, React वापरून कसे बनवावे, हे पाहुयात.
 
-आपण शेवटी काय साध्य करणार आहोत येथे पाहावे: **[Final Result](https://codepen.io/gaearon/pen/gWWZgR?editors=0010)**. जर हा कॉड तुम्हाला समजत नसेल किंवा अपरिचित असे, तर काळजी करू नका. ह्या ट्युटोरिअल चे उद्दिष्ट आपल्याला रिऍक्ट आणि त्याचे सिन्टाक्स समजणे हे आहे.
+आपण शेवटी काय साध्य करणार आहोत येथे पाहावे: **[Final Result](https://codepen.io/gaearon/pen/gWWZgR?editors=0010)**. जर हा कॉड तुम्हाला समजत नसेल किंवा अपरिचित असे, तर काळजी करू नका. ह्या ट्युटोरिअल चे उद्दिष्ट आपल्याला React आणि त्याचे सिन्टाक्स समजणे हे आहे.
```
Comment (visshaljagtap) [inline content/tutorial/tutorial.md:38]: कॉड =>कोड

Comment (visshaljagtap) [reply content/tutorial/tutorial.md:38]: सिन्टाक्स => सिंटॅक्स

File: content/tutorial/tutorial.md [hunk 4]
```diff
@@ -56,9 +55,9 @@ The tutorial is divided into several sections:
 
 हा प्रारंभ करण्याचा वेगवान मार्ग आहे!
 
-प्रथम हे नवीन टॅबमध्ये उघडा  **[Starter Code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)** नवीन टॅबमध्ये रिक्त टिक-टॅक-टू गेम बोर्ड आणि प्रतिक्रिया कोड प्रदर्शित करेल. या ट्यूटोरियल मध्ये आपण रिऍक्ट कोड संपादित करू.
+प्रथम हे नवीन टॅबमध्ये उघडा  **[Starter Code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)** नवीन टॅबमध्ये रिक्त टिक-टॅक-टू गेम बोर्ड आणि React कोड प्रदर्शित करेल. या ट्यूटोरियल मध्ये आपण React कोड संपादित करू.
 
-आपण आता दुसरा सेटअप पर्याय वगळू शकता आणि रिऍक्ट चे ओव्हरविव मिळविण्यासाठी [Overview](#overview)नवीन विभागात जा.
+आपण आता दुसरा सेटअप पर्याय वगळू शकता आणि React चे ओव्हरविव मिळविण्यासाठी [Overview](#overview)नवीन विभागात जा.
```
Comment (visshaljagtap) [inline content/tutorial/tutorial.md:60]: space before नवीन विभागात जा.

File: content/tutorial/tutorial.md [hunk 5]
```diff
@@ -118,17 +117,17 @@ import './index.css';
...
 ## ओव्हरव्हिव {#overview}
 
-आता आपण सेट केले आहे, रिऍक्टचे ओव्हरव्हिव पाहूया!
+आता आपण सेट केले आहे, React चे ओव्हरव्हिव पाहूया!
 
-### रिऍक्ट काय आहे? {#what-is-react}
+### React काय आहे? {#what-is-react}
 
-रिऍक्ट यूजर इंटरफेस तयार करण्यासाठी एक घोषणाकारक, कार्यक्षम आणि लवचिक जावास्क्रिप्ट लायब्ररी आहे. हे आपल्याला "घटक" म्हणून ओळखल्या जाणाऱ्या कोडच्या छोट्या आणि वेगळ्या तुकड्यांमधून जटिल यूआय तयार करू देते.
+React यूजर इंटरफेस तयार करण्यासाठी एक घोषणाकारक, कार्यक्षम आणि लवचिक जावास्क्रिप्ट लायब्ररी आहे. हे आपल्याला "घटक" म्हणून ओळखल्या जाणाऱ्या कोडच्या छोट्या आणि वेगळ्या तुकड्यांमधून जटिल यूआय तयार करू देते.
```
Comment (visshaljagtap) [inline content/tutorial/tutorial.md:128]: dont use घटक instead use कौम्पोनॅन्ट

File: content/tutorial/tutorial.md [hunk 6]
```diff
@@ -118,17 +117,17 @@ import './index.css';
...
 
-रिऍक्टमध्ये काही वेगवेगळ्या प्रकारचे घटक असतात, परंतु आम्ही `React.Component`उपवर्गासह प्रारंभ करू:
+React मध्ये काही वेगवेगळ्या प्रकारचे घटक असतात, परंतु आम्ही `React.Component`उपवर्गासह प्रारंभ करू:
```
Comment (visshaljagtap) [inline content/tutorial/tutorial.md:130]: घटक => कौम्पोनॅन्ट

File: content/tutorial/tutorial.md [hunk 7]
```diff
@@ -56,9 +55,9 @@ The tutorial is divided into several sections:
 
 हा प्रारंभ करण्याचा वेगवान मार्ग आहे!
 
-प्रथम हे नवीन टॅबमध्ये उघडा  **[Starter Code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)** नवीन टॅबमध्ये रिक्त टिक-टॅक-टू गेम बोर्ड आणि प्रतिक्रिया कोड प्रदर्शित करेल. या ट्यूटोरियल मध्ये आपण रिऍक्ट कोड संपादित करू.
+प्रथम हे नवीन टॅबमध्ये उघडा  **[Starter Code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)** नवीन टॅबमध्ये रिक्त टिक-टॅक-टू गेम बोर्ड आणि React कोड प्रदर्शित करेल. या ट्यूटोरियल मध्ये आपण React कोड संपादित करू.
```
Comment (visshaljagtap) [inline content/tutorial/tutorial.md:58]: टिक-टॅक-टू =>  टिक-टैक्-टो

Please change this to wherever use

File: content/tutorial/tutorial.md [hunk 8]
```diff
@@ -16,37 +16,36 @@ This tutorial doesn't assume any existing React knowledge.
...
 * [Completing the Game](#completing-the-game) हे आपल्याला **कॉमन टेक्निक** शिकण्यास मदत करेल.
-* [Adding Time Travel](#adding-time-travel) ह्याच्या साह्याने रिऍक्ट च्याअद्वितीय सामर्थ्याबद्दल आपल्याला **सखोल अंतर्दृष्टी(माहिती)** मिळेल.
+* [Adding Time Travel](#adding-time-travel) ह्याच्या साह्याने रिऍक्ट च्याअद्वितीय सामर्थ्याबद्दल आपल्याला **सखोल माहिती** मिळेल.
 
 या ट्यूटोरियलमधून मूल्य मिळविण्यासाठी आपल्याला एकाच वेळी सर्व विभाग पूर्ण करण्याची आवश्यकता नाही. आपण शक्य तितके पुढे वाचत जाण्याचा प्रयत्न करा - ते एक किंवा दोन विभाग असले तरीही चालतील.
 
 ### आपले काय बनवण्याचे उद्दिष्ट आहे? {#what-are-we-building}
 
-ह्या ट्युटोरिअल मध्ये आपण एक इंटरॅक्टिव्ह टिक-टॅक-तोव गेम, रिऍक्ट वापरून कसे बनवावे, हे पाहुयात.
+ह्या ट्युटोरिअल मध्ये आपण एक इंटरॅक्टिव्ह टिक-टॅक-तोव गेम, React वापरून कसे बनवावे, हे पाहुयात.
```
Comment (visshaljagtap) [inline content/tutorial/tutorial.md:36]: टिक-टॅक-तोव => टिक-टैक्-टो

File: content/tutorial/tutorial.md [hunk 9]
```diff
@@ -16,37 +16,36 @@ This tutorial doesn't assume any existing React knowledge.
...
 
-The tutorial is divided into several sections:
 यूटोरियल हे पुढील विभागांमध्ये विभागले गेले आहे:
 
 * [Setup for the Tutorial](#setup-for-the-tutorial) हे आपल्याला एक **स्टार्टींग पॉईंट** प्रदान करेल जेणेकरून आपण हे यूटोरियल समजू शकाल.
-* [Overview](#overview) हे आपल्याला रिऍक्ट चे **फंडामेंटल्स** कॉम्पोनन्ट्स प्रॉप्स स्टेट्स शिकवेल.
+* [Overview](#overview) हे आपल्याला React चे **फंडामेंटल्स** कॉम्पोनन्ट्स प्रॉप्स स्टेट्स शिकवेल.
 * [Completing the Game](#completing-the-game) हे आपल्याला **कॉमन टेक्निक** शिकण्यास मदत करेल.
-* [Adding Time Travel](#adding-time-travel) ह्याच्या साह्याने रिऍक्ट च्याअद्वितीय सामर्थ्याबद्दल आपल्याला **सखोल अंतर्दृष्टी(माहिती)** मिळेल.
+* [Adding Time Travel](#adding-time-travel) ह्याच्या साह्याने रिऍक्ट च्याअद्वितीय सामर्थ्याबद्दल आपल्याला **सखोल माहिती** मिळेल.
```
Comment (visshaljagtap) [inline content/tutorial/tutorial.md:30]: रिऍक्ट => React

Comment (abhaypawar) [reply content/tutorial/tutorial.md]: Done!

File: content/tutorial/tutorial.md (hunk 8 above)
Comment (abhaypawar) [reply content/tutorial/tutorial.md]: Done!

File: content/tutorial/tutorial.md (hunk 7 above)
Comment (abhaypawar) [reply content/tutorial/tutorial.md]: Done!

File: content/tutorial/tutorial.md (hunk 5 above)
Comment (abhaypawar) [reply content/tutorial/tutorial.md]: Done!

File: content/tutorial/tutorial.md (hunk 4 above)
Comment (abhaypawar) [reply content/tutorial/tutorial.md]: Done!

File: content/tutorial/tutorial.md (hunk 3 above)
Comment (abhaypawar) [reply content/tutorial/tutorial.md]: Done!

File: content/tutorial/tutorial.md (hunk 2 above)
Comment (abhaypawar) [reply content/tutorial/tutorial.md]: Done!

**Comment by visshaljagtap** [comment]
> Done. Could you Please check.
> Thanks!

LGTM! Please carry on for further translation!

**Review by visshaljagtap — APPROVED** [review]

**Pull request merged**